Packages/vcs/Lib/isofill.py

```diff
@@ -595,8 +612,9 @@ def __init__(self, Gfi_name, Gfi_name_src='default'):
             self._ext_1 = False
             self._ext_2 = False
             self._fillareastyle = 'solid'
-            self._fillareaindices = None
+            self._fillareaindices = [1, ]
             self._fillareacolors = [1, ]
+            self._fillareaopacity = []
```

Comment:
Nope. The default is different in different cases.
For solids and patterns, default opacity value is 0 and for hatches it is 100.